apartment.rb

```diff
@@ -1,2 +1,24 @@
 class Apartment
+	attr_reader :rooms, :monthly_rent
+
+	def initialize(args={})
+		@rooms = args.fetch(:rooms," ")
+		@monthly_rent = args.fetch(:monthly_rent, 1000)
+	end
+
+	def total_sqft
+		rooms.map {|room| room.sqft}.reduce(:+)
+	end
+
+	def price_per_sqft
+		self.monthly_rent / total_sqft.to_f
```

Comment:
It is actually unnecessary to include the self here because you have your attr_reader :monthly_rent, which creates the getter method for you. 
```
monthly_rent / total_sqft.to_f
```
will give you the same return value.